src/_guides/language/effective-dart/usage.md

```diff
@@ -1096,10 +1147,9 @@ class Point {
 {% endprettify %}
 
 This `this.` syntax before a constructor parameter is called an "initializing
-formal". You can't always take advantage of it. In particular, using it means
-the parameter is not visible in the initialization list. But, when you can, you
-should.
-
+formal". You can't always take advantage of it. Sometimes you want to have a
+named parameter whose name doesn't match the name of the field you are
+initializing. But, when you can use initialing formals, you should.
```

Comment:
But, -> But
maybe, for emphasis and ease of reading: can -> _can_   (and should -> _should_)
initialing -> initializing